api/client/diff.go

```diff
@@ -34,7 +34,7 @@ func (cli *DockerCli) CmdDiff(args ...string) error {
 		var kind string
 		switch change.Kind {
 		case archive.ChangeModify:
-			kind = "C"
+			kind = "M"
```

Comment:
I have looked the command "docker diff --help" it shows me Usage: docker diff [OPTIONS] CONTAINER, Inspect changes on a container's filesystem. That is to say the change will not influence the guide line, and we can know that the "diif.go" is just a independent api function, so it will not affect others or the other parts of the docker project. Waiting for your reply!